app.cantara.Cantara.yml

```diff
@@ -0,0 +1,40 @@
+app-id: app.cantara.Cantara
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: cantara
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
```

Comment:
I can chime in here, as I helped @reckel-jm write this flatpak yml (It's also my first flatpak package, so correct me if I'm wrong).

Cantara needs to open whole directories instead of just files. According to the flatpak portal documentation (https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.FileChooser) this functionality was added in version 3 of the `org.freedesktop.portal.FileChooser` portal.
According to https://docs.flatpak.org/en/latest/portals-qt.html Qt supports this functionality starting with Qt 5.18.90 (There is no qt 5.18.90. They probably mean xdg-desktop-portal-kde 5.18.90).

Looking at the patch to Qt (https://github.com/qt/qtbase/commit/844967d297327dc72a1aa67644c1b2fa3fe6839c) shows that it didn't land in time for Qt 5.15.

For this reason this permission is required until Cantara can be updated to use Qt6 (which in turn depends on lazarus/freepascal supporting Qt6).